src/lib/getStoredConfig.ts

```diff
@@ -0,0 +1,13 @@
+import configstore from './configstore';
+
+/**
+ * Retrieves stored user data values from env variables or configstore,
+ * with env variables taking precedent
+ */
+export default function getStoredConfig(): { apiKey?: string; email?: string; project?: string } {
```

Comment:
The function name `getStoredConfig()` kinda implies that it's only pulling from the `configstore` (specifically because environment variables are ephemeral and not necessarily _stored_ anywhere). Could you rename this file + function to `getCurrentConfig` or just `getConfig`?

Reply:
+1 to `getConfig` too

Reply:
I went back and forth because we also have this `config` also lol... I like `getCurrentConfig`!:

https://github.com/readmeio/rdme/blob/50996ae9072db66614f72976bdeb945c20e4eb42/config/default.js#L1-L6

Reply:
works for me

Reply:
done! https://github.com/readmeio/rdme/pull/709/commits/0fdcc978fec4b49c21a15590c1501df5dd0905c3